src/visualizers/VisualizerHistogram.ts

```diff
@@ -0,0 +1,211 @@
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import {VisualizerDefault} from './VisualizerDefault'
+import {natlog} from '../shared/math'
+
+class HistogramVisualizer extends VisualizerDefault {
```

Comment:
Needs documentation comments, that will automatically show up in the on-line user guide. See VisualizerDifferences for a template.

Also, this would be an appropriate time to make the name of the visualizer more specific: it only histograms the number of factors, so the name "Histogram" is a little misleading, especially if the user were looking to histogram the values of a sequence. If in a future PR you added the ability to histogram either the values of a sequence or the number of factors of the values of a sequence (which wouldn't be hard to do), then you could change the name back to Histogram. But at this point, I would not recommend adding that ability in this PR, so for now, please just change the name of the visualizer. Thanks!